controllers/cloud.redhat.com/providers/kafka/provider.go

```diff
@@ -22,6 +23,28 @@ func GetKafka(c *p.Provider) (p.ClowderProvider, error) {
 	}
 }
 
+func getKafkaNamespace(e *crd.ClowdEnvironment) string {
```

Comment:
I'm a little reticent about putting mode specific knowledge in the top level package

Reply:
I'll change this by having the provider implementation tell `getConnectNamespace` which default value to use